ui/submodules/packages/components/src/components/usages/filters.tsx

```diff
@@ -23,7 +23,7 @@ export function Filters(): JSX.Element {
                 value={value}
                 onChange={onChange}
                 sx={{ pl: 1, flex: 1 }}
-                placeholder={t("table.filter.QUICK", "Filter...")}
+                placeholder={t("table.filter.QUICK", "Search...")}
```

Comment:
To chyba coś nie działa, tzn  w innym miejscu zmiana jest potrzebna? U mnie nadal placeholder to `Filter...`